packages/config/config/devices/0x0234/zif5028_heatit_z-din_616.json

```diff
@@ -293,7 +464,7 @@
 			"defaultValue": 0,
 			"readOnly": false,
 			"writeOnly": false,
-			"allowManualEntry": false,
+			"allowManualEntry": true,
 			"options": [
 				{
 					"label": "Central Scene notifications enabled. (Standard)",
```

Comment:
Change these labels to `"Enabled"` and `"Disabled"` please

Reply:
all of the labels or just "allowManualEntry" ?
Documentation fetched from openHAB with the script provided by you zwavejs.
Do you have a specific guideline for these devices, this is unclear to me ? and where to find this documentation ?

Reply:
This only refers to the labels. We are trying to clean up all options in all ~2500 config files so they have a standardized language to them. Part of this is to formulate the labels for these kinds of parameters in a way that the options can be just "Enabled" and "Disabled" (and later be converted to booleans).

Reply:
Since you seem to confuse `allowManualEntry` `true/false`: `allowManualEntry` should be `true` if other values beside the given options should be possible to enter. This is not the case here (and in some of the other parameters).

Reply:
![image](https://user-images.githubusercontent.com/33527165/111028369-81ee3e80-83f6-11eb-8050-4825bdec4ced.png)